xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -1003,12 +1143,20 @@
         <Parameter Name="module" Type="System.Reflection.Metadata.ModuleReferenceHandle" Index="3" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="method">To be added.</param>
-        <param name="attributes">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="module">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="method">The method definition handle.</param>
+        <param name="attributes">The method import attributes.</param>
+        <param name="name">The unmanaged method name.</param>
+        <param name="module">The module containing the unmanaged method.</param>
+        <summary>Adds import information to a method definition (P/Invoke).</summary>
```

Comment:
Not clear what P/Invoke signifies.

Reply:
I agree, not clear. It seems that `method` is a handle to a method that can be P/Invoked from an external DLL. I'll remove the word, since the data type should explain that.
```suggestion
        <summary>Adds import information to a method definition.</summary>
```